src/main/java/xyz/aadev/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -241,8 +281,12 @@ public void burnTime(int i) {
         }
     }
 
+
     @Override
     public void update() {
+
+        //System.out.println(sides[0]+" "+worldObj.isRemote);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)